src/utilities/message/messageParser.ts

```diff
@@ -9,9 +9,20 @@ export class MessageParser {
         errors : []
       }
     };
+    const reg = /^https?:\/\//i;
+    let url = response.requestMessage.url;
+    if (!reg.test(response.requestMessage.url)) {
+      url = `${response.requestMessage.baseUrl}/${url}`;
+    }
 
-    if (response.content.errors || response.content.message) {
-      var errors = response.content.errors || [response.content.message];
+    if (response.statusCode === 401 || response.statusCode === 403) {
+      result.response.message = `Niet bevoegd (${response.statusCode})`;
+      result.response.errors = [
+        `U hebt niet voldoende rechten om deze data op te halen: ${url}`
+      ]
+    }
+    else if (response.content.errors || response.content.message) {
+      let errors = response.content.errors || [response.content.message];
```

Comment:
Controleer anders nog in restMessage.ts nog of title en message niet gelijk zijn zodat ge niet 2x hetzelfde bericht krijgt in de toast

Reply:
ja ik vond dees ook funky code, ma wou eraf blijven omdak geen idee heb wa er zal breken dan (heb hier enkel de 'var' vervangen door 'let')
zal is zien